lib/domain.js

```diff
@@ -29,12 +29,12 @@
 const util = require('util');
 const EventEmitter = require('events');
 const {
-  ERR_DOMAIN_CALLBACK_NOT_AVAILABLE,
-  ERR_DOMAIN_CANNOT_SET_UNCAUGHT_EXCEPTION_CAPTURE,
   ERR_UNHANDLED_ERROR
 } = require('internal/errors').codes;
 const { createHook } = require('async_hooks');
 
+const CAPTURE_CB_KEY = Symbol('domain');
```

Comment:
Can we do something similar to what https://github.com/nodejs/node/blob/master/doc/guides/using-symbols.md suggests? e.g.

```suggestion
const kCaptureKeyDomain = Symbol('kCaptureKeyDomain');
```

Having identical names for the variable and the symbol itself helps with grepping :)

Reply:
Great suggestion, I'll do that 👍